engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -808,9 +810,15 @@ public void postSave() {
     public void shutdown() {
     }
 
+    @Override
+    public RenderGraph getRenderGraph() {
+        return renderGraph;
+    }
+
     @Command(shortDescription = "Debugging command for DAG.", requiredPermission = PermissionManager.NO_PERMISSION)
     public void dagNodeCommand(@CommandParam("nodeUri") final String nodeUri, @CommandParam("command") final String command, @CommandParam(value= "arguments") final String... arguments) {
-        Node node = renderGraph.findNode(nodeUri);
+        WorldRenderer worldRenderer = CoreRegistry.get(WorldRenderer.class);
```

Comment:
Since renderGraph is no longer a static variable, the instance of WR which is handling the commands (which was created by RegisterSystem) does not have the correct renderGraph variable. So, I'm obtaining it via CoreRegistry (rather than have a `@In WorldRenderer worldRenderer` inside WorldRenderer class - no need to confuse people so much. Might I recommend reconsidering the creation of a dedicated NodeCommands class? A `@In WorldRenderer worldRenderer` won't confuse people in that class.

Reply:
I need a bit more context on why there are two separate instances of WorldRenderer, see earlier comment.